gdal/frmts/prf/phprfdataset.cpp

```diff
@@ -55,21 +55,21 @@ class PhPrfBand : public VRTSourcedRasterBand
         osOverview.push_back( ov );
     }
 
-    int GetOverviewCount()
+    int GetOverviewCount() CPL_OVERRIDE
```

Comment:
you can just use override, even in non C++11 mode (in that case it will be an empty #define)